R/utils-custom.R

```diff
@@ -12,22 +12,47 @@
 theme_tiltplot <- function() {
   theme_classic() +
     theme(
-      plot.title = element_text(hjust = 0.5, size = 16),
-      axis.title = element_text(size = 12),
-      axis.text = element_text(size = 10),
-      legend.title = element_text(size = 12),
-      legend.text = element_text(size = 10),
+      plot.title = element_text(hjust = 0.5, size = 16, family = tilt_headline_font()),
+      strip.text = element_text(family = tilt_text_font()),
+      axis.title = element_text(size = 10, family = tilt_text_font()),
+      axis.text = element_text(size = 10, family = tilt_text_font()),
+      legend.title = element_text(size = 12, family = tilt_text_font()),
+      legend.text = element_text(size = 10, family = tilt_text_font()),
       legend.position = "bottom",
       panel.grid.major.y = element_blank(),
       panel.grid.minor.y = element_blank(),
       panel.spacing.y = unit(0.5, "lines")
     )
 }
 
-score_colors <- function(...) c("low" = "#007F00", "medium" = "#FFC300", "high" = "#FF5733")
+fallback_font <- function() "Open Sans"
+
+tilt_text_font <- function() ifelse(exists("Roboto"), fallback_font(), "Roboto")
+
+tilt_headline_font <- function() ifelse(exists("Roboto Condensed"), fallback_font(), "Roboto Condensed")
+
+load_custom_font <- function() {
+  font_add_google(tilt_text_font(), tilt_text_font())
+  font_add_google(tilt_headline_font(), tilt_headline_font())
+  font_add_google(fallback_font(), fallback_font())
+
+  showtext_auto()
+}
```

Comment:
Thanks ! I do not need it anymore since I have change to the use of a vanilla font instead of the more rare "Roboto" font. It is noted though, thanks !